README.md

```diff
@@ -105,11 +105,18 @@ Note that IDP IDs (`other-google` and `other-orcid` in the example above) must b
 Also note that the OIDC clients you create must be granted `read-storage` access to all the data in the external
 Data Commons via the data-commons' `user.yaml`.
 
+The `id_token_username` property for OIDC clients can be configured with `.` in between strings for a nested username inside a token.
+For example if the token jwt has username encoded in the json as `token["context"]["user"]["name"]`.
+We can write this in the paramaters as `"id_token_username": "context.user.name"`
+The default if nothing is specified for a fence client it defaults to `"context.user.name"` for a non-fence client the default is `"email"`
```

Comment:
it displays as a single paragraph, we need to either add newlines to display multiple paragraphs or add periods to separate the sentences:

![Screenshot 2024-05-09 at 10 07 56 AM](https://github.com/uc-cdis/workspace-token-service/assets/4224001/e8edd6c3-f700-42af-9ef2-2b23671304c8)

Reply:
```suggestion
The `id_token_username` property for OIDC clients can be configured with `.` in between strings for a nested username inside a token.
For example if the token jwt has username encoded in the json as `token["context"]["user"]["name"]`:
We can write this in the parameters as `"id_token_username": "context.user.name"`.
If nothing is specified, for a fence client the default is `"context.user.name"`, for a non-fence client the default is `"email"`.
```